locale/en.po

```diff
@@ -2093,10 +2095,10 @@ msgstr ""
 #: ../src/gui_common/tooltip/ToolTipManager.tscn:1716
 msgid "TOXIN_VACUOLE_PROCESSES_DESCRIPTION"
 msgstr ""
-"Turns [thrive:compound]atp[/thrive:compound] into [thrive:"
-"compound]oxytoxy[/thrive:compound]. Rate scales with concentration of "
-"[thrive:compound]oxygen[/thrive:compound]. Can release toxins by pressing "
-"[thrive:input]g_fire_toxin[/thrive:input]."
+"Turns [thrive:compound type=\"atp\"]ATP[/thrive:compound] into [thrive:"
+"compound type=\"oxytoxy\"]OxyToxy NT[/thrive:compound]. Rate scales with "
```

Comment:
I think with the type already specified, the text inside the bbcode tags should be left empty *unless* it requires to be customized from the normal organelle name.

Reply:
So I'm assuming it'd look like this `[thrive:compound type="atp"][/thrive:compound]`, would that be valid and not cause warnings in Weblate?

Reply:
I think it should be fine, that's valid bbcode syntax with nothing inside the tag.
The current warning is about the tag definition changing between the translation and source language.